src/Atmos/Model/moisture.jl

```diff
@@ -15,27 +15,40 @@ end
 function gradvariables!(::MoistureModel, transform::Vars, state::Vars, aux::Vars, t::Real)
 end
 
-# TODO: rewrite to use MoistThermo
-struct DryModel <: Atmos.MoistureModel
+function internal_energy(m::MoistureModel, state::Vars, aux::Vars)
```

Comment:
Wouldn't `MoistThermodynamics` be a better place for this function?

Reply:
This is just a wrapper function to help facilitate computing internal energy with a simple interface (since MoistThermo has total energy, and does not have kinetic and potential energy).